package.json

```diff
@@ -14,16 +14,17 @@
     "start:client": "npm start --prefix development/client",
     "start:server": "npm start --prefix development/server",
     "start:dev": "run-p start:client start:server",
-    "dist": "tsx build.ts",
+    "dist:chrome": "npm run build -- --env TARGET=chrome && tsx build.ts chrome",
+    "dist:firefox": "npm run build -- --env TARGET=firefox && tsx build.ts firefox",
     "build": "npm run clean && webpack --env NODE_ENV=production --progress",
     "build:dev": "npm run clean && webpack --env NODE_ENV=development --progress",
     "clean": "rimraf build && mkdir build && rimraf *.zip",
-    "changeset-publish": "npm run build && changeset publish",
+    "changeset-publish": "npm run build -- --env TARGET=firefox && changeset publish",
```

Comment:
Curious about `TARGET=firefox` here - I'm guessing it's just easier to ship just the FF artifacts to GitHub/npm? We may want to make the command name more specific to indicate that.

Reply:
I added a check to ensure [`TARGET` is defined](https://github.com/apollographql/apollo-client-devtools/pull/1225/files#diff-504be09670485a92ff6bcb692041129e877eda9058a8c3114c42a53a0e0d1194R25-R27) as a means to make builds more intentional as to what we are building. This was also a bit of future-proofing. At some point I assume we'll want to support Edge and Safari and the `TARGET` makes it obvious what we are building.

Previous builds sort of already implicitly built it as the "firefox" target (though there really were no differences with the chrome build until now). My thinking here was that today it's build with manifest v2 which is now the firefox manifest, so this maintains compatibility with how its built today.

Thoughts? Would you prefer we default to a target if one isn't specified?

Reply:
> Previous builds sort of already implicitly built it as the "firefox" target

Previous builds weren't aware of any target afaik unless running in development mode since `WebExtPlugin` was the only one consuming `env.TARGET`, but maybe there's something I'm missing here?

Reply:
Ya that's correct. I probably used the term "firefox target" way too loosely 😂. `WebExtPlugin` was the only one that consumed it.

Do you know how the manifest file is used in the npm package outside of the extension? I'm actually a bit unsure how the npm package works on its own. Perhaps this is a good topic for our 1:1 today 🙂

Reply:
Sounds good :) Yeah, the manifest itself should only matter when shipping to an extension store.

There was a question about running it as a standalone package like Vue allows https://github.com/apollographql/apollo-client-devtools/discussions/1046. I think we'll likely have to make some changes to support that (the npm package has a few hundred DLs/week so perhaps it's already possible, I haven't checked), but let's handle that in a separate PR regardless.